tasks/validate.js

```diff
@@ -108,7 +108,7 @@ module.exports = function(grunt) {
 
   var reviewed = async function() {
     // Do all books have a valid reviewer?
-    var noReviewer = grunt.data.shelf.filter(b => !(b.reviewer in grunt.data.json.reviewers));
+    var noReviewer = grunt.data.shelf.filter(b => !(b.reviewer_array[0] in grunt.data.json.reviewers || b.reviewer_array[1] in grunt.data.json.reviewers));

```

Comment:
Again, this will work fine for 2 reviewers but not for more. Let's code for something more sustainable. One thing that will be useful is the `Array.every` function: 
```js
var noReviewer = grunt.data.shelf.filter(b => !b.reviewers.every(r => r in grunt.data.json.reviewers));
```
You'll also want to update the logging code below so that it spits out the right entry.